Marlin/Configuration_adv.h

```diff
@@ -861,6 +878,7 @@
   #define PAUSE_PARK_XY_FEEDRATE 100          // X and Y axes feedrate in mm/s (also used for delta printers Z axis)
   #define PAUSE_PARK_Z_FEEDRATE 5             // Z axis feedrate in mm/s (not used for delta printers)
   #define PAUSE_PARK_RETRACT_FEEDRATE 60      // Initial retract feedrate in mm/s
+  #define PAUSE_PARK_RECOVER_FEEDRATE 20      // Resuming recover feedrate in mm/s
```

Comment:
Need , to have a sure recovery when return to print
In reality , the recovery is millimetric , and is different when feedrate change , nozzle size , temp , and more
But , in reality too , a ' 15/30 mms ' feedrate works fine in all situations